### ISSUE TITLE
Unable to select origin

### ISSUE THREAD

**Issue by Metropolis0000** [sakisiil/Origin-Datapacks#1]
I can't get the datapack to work. I have everything installed right from what I know, and it even says that the datapack is enabled when I check in game, but I can't actually select the origin. I tried using the origin orb and completely wiping my origin, but it didn't show up at all. Is there any way to fix this?

**Comment by Mrpiggy9x** [comment]
That happened to me as well, it still shows the other bee origin instead of the bumblebee one

**Comment by Coreycore420** [comment]
I used to have this problem are you sure I didn't put it in the mod folder instead of your worlds data pack folder? and are you using the new bee origin file?

(sorry I can't quite remember what I did to fix it)

**Comment by alyxei** [comment]
I'm having the same issue but mine is in my mods folder

**Comment by Coreycore420** [comment]
Make sure that it’s definitely in the mods folder

**Comment by Phenicwashere** [comment]
yeah i have the same problem, i am using the old bee file though, would that effect anything?

**Comment by alyxei** [comment]
Okay so first you need to download the original origins mod as well as fabric api and launcher. Then download the bumblebee datapack and don't put it in the mods folder. When you create a new world, there is a button that says "data packs" and you just click on that and either open your packs folder and drag the datapack into there, or drag the zip file directly into minecraft.

**Comment by Phenicwashere** [comment]
@alyxei  ok i followed these directions and the bee origin worked but the racoon didn't

**Comment by alyxei** [comment]
@Phenicwashere you just do the same but with the raccoon file

**Comment by Phenicwashere** [comment]
@alyxei  i did but it didn't work
